SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Build/Build.cs

```diff
@@ -21,6 +21,25 @@ shift 1
 chmod +x $SCRIPT
 gosu $NEW_USER ""${{SCRIPT}}"" ""$@"" 2> >(grep -v xdg-user-dir >&2)";
 
+
+        // This is for internal use only. We do not support Linux clients.
+        private const string FakeClientWorkerShellScript =
+			@"#!/bin/bash
```

Comment:
```suggestion
@"#!/bin/bash
```

Reply:
The other worker shell script also had this line indented like this (at the top of the file). Should I un-indent that one as well?

Reply:
eeeeeeehhhhhhhhhhhhhHHHHHHHHHHHHHHHHHHHHH probably

Reply:
this is now done